src/api/polls.js

```diff
@@ -4,6 +4,9 @@ const POLLS_API_URL =
 
 export async function getPoll(pollId) {
   const response = await fetch(`${POLLS_API_URL}/${pollId}`);
+  if (!response.ok) {
+    throw new Error(response.statusText);
```

Comment:
I noticed that the `statusText` is an empty string `""` if the poll doesn't exists and the fetch returns a 404 status.
This is the reason why https://umfragy-git-errormessage.meloneminze.now.sh/polls/7/vote is not showing an error message.
I think a good solution is to add the status code to the error message too:
```js
throw new Error(`${response.status} ${response.statusText}`);
```

Or only return the status code and show custom error messages.

I have the same issue in my poll app.